src/app/work-item/work-item.service.ts

```diff
@@ -24,7 +24,12 @@ export class WorkItemService {
 
   getWorkItem(id: number): Promise<WorkItem> {
     return this.getWorkItems()
-      .then(workItems => workItems.find(workItem => workItem.id === id));
+      .then((workItems) => {
+ 		  // workItem.id is string 
+		  // but id is number 
+		  // so we use == instead ===
+		  return workItems.find(workItem => workItem.id == id);
+	  });
```

Comment:
Good find. 
In an arrow function, if you don't do anything else in the braces you do not need the 'return'. It is easier to read, when using arrow functions, on one line. I would not make it multiple lines with a return just to add comments. If you think the comments are needed then add them above the '.then'. I do think the comment is needed since the switch from === to == needs be explained.